Dockerfile

```
@@ -8,31 +8,39 @@ ENV DEBIAN_FRONTEND=noninteractive \
 
 RUN apt-get -qq update \
   && apt-get install -y --no-install-recommends \
-    apt-transport-https \
-    build-essential \
-    curl \
-    dirmngr \
-    gnupg \
-    git \
-    ruby-full \
-    twine \
+  apt-transport-https \
+  build-essential \
+  curl \
+  dirmngr \
+  gnupg \
+  git \
+  ruby-full \
+  twine \
+  jq \
+  unzip \
   && curl -fsSL https://packages.microsoft.com/config/debian/10/packages-microsoft-prod.deb -o /tmp/packages-microsoft-prod.deb \
   && dpkg -i /tmp/packages-microsoft-prod.deb \
   && rm /tmp/packages-microsoft-prod.deb \
   && echo 'deb [arch=amd64] https://download.docker.com/linux/debian buster stable' >> /etc/apt/sources.list \
   && curl -fsSL https://download.docker.com/linux/debian/gpg | apt-key add - \
   && apt-get update -qq \
   && apt-get install -y --no-install-recommends \
-    dotnet-sdk-3.1 \
-    docker-ce-cli \
+  dotnet-sdk-3.1 \
```

Given the image has .NET already (assuming it's because of `nuget` target), I can have a build of SymbolCollector that's 'runtime-dependant' which should reduce its size to just a few KBs.

Also, can we bump to 5.0?

```suggestion
  dotnet-sdk-5.0 \
```

---

Opened another PR for that: https://github.com/getsentry/craft/pull/267